src/main/java/org/sonarlint/intellij/notifications/SonarLintProjectNotifications.java

```diff
@@ -113,12 +113,25 @@ public void notifyWiderLanguageSupport(String language) {
       TITLE_SONARLINT_SUGGESTIONS,
       "Enable " + language +" analysis by connecting your project",
       NotificationType.WARNING);
+    addNotifications(notification);
+    notification.notify(myProject);
+  }
+
+  public void notifyAdvancedLanguageSupport(String language) {
+    var notification = IN_CONTEXT_PROMOTION_GROUP.createNotification(
+      TITLE_SONARLINT_SUGGESTIONS,
+      "Enable advanced " + language + " rules by connecting your project",
+      NotificationType.WARNING);
```

Comment:
I missed it in the first PR but I think a INFO type would be more adapted for those notifications